### PR TITLE
added aadii0408.json

### PR DESCRIPTION
**Pull request by aadii0408**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4004"></a>

### CODE REVIEW

**Comment by aadii0408** [comment]
Hey, Please review this @eddiejaoude @Pradumnasaraf

File: data/aadii0408.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  {
```
Comment (stevenovak123) [inline data/aadii0408.json:6]: Please follow the Read me for the Json file. tags dont need to be in a separate block

File: data/aadii0408.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"]
+  }
+  "avatar": "https://github.com/aadii0408.png"
```
Comment (stevenovak123) [inline data/aadii0408.json:9]: add a comma at the end of line

File: data/aadii0408.json (hunk 1 above)
Comment (stevenovak123) [reply data/aadii0408.json:6]: 1. https://linkfree.eddiehub.io/docs/quickstart
2. https://github.com/EddieHubCommunity/LinkFree/blob/main/data/eddiejaoude.json

Link 1 will give you a visual demo.
Link 2 will show you the format of how the json has to be.

File: data/aadii0408.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  {
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"]
```
Comment (stevenovak123) [inline data/aadii0408.json:7]: Comma at the end of the line

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

**Review by aasthashahcodes — CHANGES_REQUESTED** [review]
You may want to delete the curly brackets on Line 6 & 8. Tags don't need another block. 

Eg. 
"name": "Aastha Shah",
  "displayStatsPublic": true,
  "type": "personal",
  "bio": "Newbie in tech | Learning Web-Dev | Learning from Communities",
  "tags": ["Communities", "Frontend", "Beginner"],
  "avatar": "https://github.com/AasthaShahCodes.png",

**Comment by aasthashahcodes** [comment]
Also, you may need to add commas after the end of 7th & 9th lines.

**Review by iamthecloverly — APPROVED** [review]

**Comment by aadii0408** [comment]
Hey, Please review this @eddiejaoude  @Pradumnasaraf

File: data/aadii0408.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"],
+  
+  "avatar": "https://github.com/aadii0408.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aadii0408",
```
Comment (kumarsonsoff3) [inline data/aadii0408.json:13]: It seems like you've added a wrong URL as this account doesn't exists. Please update it with the correct one! 🛠

image

**Review by kumarsonsoff3 — DISMISSED** [review]

File: data/aadii0408.json (hunk 4 above)
Comment (Pradumnasaraf) [reply data/aadii0408.json:13]: It's not a PR blocker. Then can raise another PR to fix that. 

Thank you

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**